ex11_html-css-advanced

Indigoinspn opened the pull request:
task-01.html  Applying mask to rounded image
task-02.html  Three CSS animation.
task-03.html  Floating label to the input field.
task-04.html  Side bar of the landing page.

Indigoinspn committed:
ex11_html-css-advanced
task-01.html  Applying mask to rounded image
task-02.html  Three CSS animation.
task-03.html  Floating label to the input field.
task-04.html  Side bar of the landing page.

rakotoz commented on src/ex11_html-css-advanced/task-01.html:
```diff
@@ -0,0 +1,39 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>task-01.html - Round image with mask </title>
+    
+    <style>
```
Стили должны быть в отдельном файле

Indigoinspn replied:
Вынесла

rakotoz commented on src/ex11_html-css-advanced/task-03.html:
```diff
@@ -0,0 +1,85 @@
+            
+    </form>
+
+    <script>
```
Скрипты выноси в отдельный файл

Indigoinspn replied:
Вынесла

rakotoz commented on src/ex11_html-css-advanced/task-04.html:
```diff
@@ -0,0 +1,167 @@
+            <!--  ******** Side bar SECOND BLOCK *********** -->
+
+            <div id="second-block">
+                <div> <i class="fas fa-book"></i> Now Reading </div>
```
Это меню, тут должны быть ссылки

Indigoinspn replied:
Заменила на ссылки.

Indigoinspn committed:
Move css rules and js to separate files.
Change id and tag selectors to class selectors.
Task-04: div navigation items replace with link tags.
